templates/emails/enroll_email_allowedsubject.txt

```diff
@@ -0,0 +1,5 @@
+<%! from django.utils.translation import ugettext as _ %>
+
+${_("""Invitation to register for Enthought's "{course_name}" """.format(
+    course_name=course.display_name_with_default
+))}
```

Comment:
Nope, we need to maintain the same name as what edx uses otherwise template overriding won't work. Sadly, they use the badly formatted name. :(